src/Raven.Client/Exceptions/Database/IndexCompactionInProgressException.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
Do we need this exception in the `Client`?

Reply:
Yes

Reply:
Yes, Do you want me to use another exception instead?

Reply:
No. Just moved to the server.

Reply:
What about this?